go.mod

```diff
@@ -7,79 +7,99 @@ go 1.23
 // where the IsZero method on pointers that implement the IsZeroer interface is not invoked.
 replace github.com/go-openapi/strfmt => github.com/go-openapi/strfmt v0.21.5
 
+// Copied from the Chronosphere terraform provider go.mod file. Can be removed once the version of Chronosphere
+// terraform provider required below no longer pins the version of the terraform plugin SDK.
+replace github.com/hashicorp/terraform-plugin-sdk/v2 v2.10.1 => github.com/chronosphereio/terraform-plugin-sdk/v2 v2.0.0-20220830001851-edb6d12500f2
+
 require (
 	github.com/alecthomas/participle/v2 v2.1.0
 	github.com/anyascii/go v0.3.2
+	github.com/chronosphereio/terraform-provider-chronosphere v1.6.2
```

Comment:
(no action needed) this ends up pulling in a fair few new deps. not a huge deal, but i wonder if we should move the conversion functions out of the `chronosphere` package into a subpackage